src/Workspaces/Core/Portable/RegularExpressions/RegexKind.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.RegularExpressions
+{
+    internal enum RegexKind
```

Comment:
Note: should proably be called SyntaxKind.  But i figured Roslyn devs would be annoyed if searching for SyntaxKind in something like NavigateTo now showed this as well.